doc/sphinx/documentation/tutorials/small_molecule_tutorial.rst

```diff
@@ -55,54 +55,47 @@ Indexing
-This duplicates the models for the individual components rather than sharing them, which allows greater flexibility in refinement (and is critical for scan varying refinement).
+Provided that the lattices overlap in the reciprocal lattice view above, the indexing should be straightforward and will guarantee that all lattices are consistently indexed as a single matrix is used.
 
 .. code-block:: bash
 
    dials.index imported.expt strong.refl
 
-Without any additional input, the indexing will determine the most approproiate primitive lattice parameters and orientation which desctibe the observed reciprocal lattice positions.
+Without any additional input, the indexing will determine the most appropriate primitive (i.e. triclinic) lattice parameters and orientation which desctibe the observed reciprocal lattice positions.
 
 Bravais lattice determination
 -----------------------------
 
-In the single sequence tutorial the determination of the Bravais lattice is performed between indexing and refinement.
-This step however will only work on a single lattice at a time.
-Therefore in this case the analysis will be performed, the results verified then the conclusion fed back into indexing as follows:
+Prior to indexing the choice of Bravais lattice may be made, though this is entirely optional and all processing may be performed with a triclinic lattice:
 
 .. code-block:: bash
 
-   dials.refine_bravais_settings indexed.refl indexed.expt crystal_id=0
-   dials.refine_bravais_settings indexed.refl indexed.expt crystal_id=1
-   dials.refine_bravais_settings indexed.refl indexed.expt crystal_id=2
-   dials.refine_bravais_settings indexed.refl indexed.expt crystal_id=3
+   dials.refine_bravais_settings indexed.refl indexed.expt
 
-Inspect the results, conclude that the oP lattice is appropriate then assign this as a space group for indexing (in this case, P222)
+Once this has run you can manually rerun indexing with:
 
 .. code-block:: bash
 
    dials.index imported.expt strong.refl space_group=P222
 
-This will once again consistently index the data, this time enforcing the lattice constraints.
+to assign the lattice, or manually reindex the data to match setting #5 (though in this case that is a no-op) - or as mentioned above proceed with the lattice unconstrained. 
 
 Refinement
 ----------
 
-Prior to integration we want to refine the experimental geometry and the scan varying crystal orientation and unit cell. This is performed in two steps — the first is to perform static refinement on each indexed sequence, the second to take this refined model and refine the unit cell and orientation allowing for time varying parameters:
+Prior to integration we want to refine the experimental geometry and the scan varying crystal orientation and unit cell: in the refinement however the refinement is done first with a scan static model, then with scan varying (which allows for small variations in the sample orientation etc.)
```

Comment:
Clarity decreased for me here.

Reply:
Attemtpted to improve text (agree clarity reduced above)